dir.targets

```diff
@@ -110,4 +110,15 @@
     <Message Importance="High" Condition="'$(ConfigurationErrorMsg)' != ''" Text="$(MSBuildProjectFullPath) ConfigurationErrorMessage: $(ConfigurationErrorMsg)" />
   </Target>
 
+  <Target Name="GenerateReferenceAssembly">
```

Comment:
Should we hook this target up to the build so it will start to pend changes if any APIs change and hopefully we don't accidentally forget to update a ref or someone update it manually?